### ISSUE TITLE
[FEAT.] : Show installed mods that aren't listed on BeatMods

### ISSUE THREAD

**Issue by beaudamion** [Zagrios/bs-manager#469]
## Problem
Manually installed mods that aren't listed on BeatMods do not show up in the mods list.

## Solution
A "Other Mods" Section at the bottom of the list after BeatMods would be nice Showing the name and version number.

**Comment by Fatalution** [comment]
Yes please. The same also applies to the unlisted versions of the mods (i.e.you have compiled another version of Chroma) - as currently the Mods tab would show that mod uninstalled (in this case "Chroma" would be unchecked even though it is present in Plugins folder)
